.github/dependabot.yml

```diff
@@ -0,0 +1,14 @@
+version: 2
+updates:
+  - package_manager: "gomod"
+    directory: "/"
+    vendor: false
+    schedule:
+      interval: "weekly"
+    labels: ["dependencies"]
+  - package_manager: "docker"
```

Comment:
I do not see any rule about busybox SHA update rule, is it a magic?

Reply:
It actually works with digest, but not sure how it can detect ARM/AMD tbh, so that will not for for now. Still I want to try this path (:

Reply:
let's try it :)